microservices/frontend/src/components/app/App.js

```diff
@@ -12,9 +12,17 @@ import { ConfirmationPage } from '../../modules/login/ConfirmationPage';
 import { AdminPage } from '../../modules/admin/AdminPage';
 import { RestorePage } from '../../modules/login/RestorePage';
 import { ToastContainer } from 'react-toastify';
+import { useState } from 'react';
+import React from 'react';
 
 export function App () {
   const { loading, user } = useStoreWithInitializer(({ app }) => app, load);
+  const [parentCallBack, setParentCallBack] = useState(false)
+
+  const onTrigger = (event) => {
```

Comment:
there's no need to do this on the App components We should handle this as part of the App reducer. (the state to display or not the modal)